editorial_workflow/EDITORIAL.R

```diff
@@ -0,0 +1,38 @@
+# Clean up previous PR if necessary
+usethis::pr_finish()
+
+# Fetch current PRs interactively
+usethis::pr_fetch()
+pr <- 830             # <--- Update PR number when you get it
+pkg <- FALSE          # <--- Is this a post about a Peer-Reviewed Package?
+
+# Get the relevant md file (and check)
+(b <- gert::git_diff("HEAD^")$new |> stringr::str_subset("\\.md$"))
```

Comment:
just curious: how does this work if there's also an author file?

Reply:
(but I really like this strategy! I'd have thought of the API first, using Git is so much quicker)

Reply:
That's a good point! I'll have to test it out!